### ISSUE TITLE
Would it be possible to get an example of combined .js and .ts in the same project

### ISSUE THREAD

**Issue by scamden** [TypeStrong/tsify#151]
It's not clear where the compiled js gets written in tsify and I'm unable to require the result of a compiled ts module from a js one. Is there a straightforward way to do this? Would love a simple example.

**Comment by scamden** [comment]
hmm i saw your test case with jsRoot but when i used that exact syntax, the module i require is found but is still the empty module.exports object...

**Comment by NeverwinterMoon** [comment]
I think I am having exactly the same issue (https://github.com/TypeStrong/tsify/issues/152). My problem disappears if I use version 0.15.3. I don't use any additional configuration, just all the defaults.

**Comment by scamden** [comment]
Ya ours disappeared at 0.14.0, will try the version you mentioned also,
just for info next week
Sterling Camden
Sent from SalesforceIQ

On July 2, 2016 at 2:18 AM, NeverwinterMoon<notifications@github.com> wrote:

I think I am having exactly the same issue (#152
<https://github.com/TypeStrong/tsify/issues/152>). My problem disappears if
I use version 0.15.3. I don't use any additional configuration, just all
the defaults.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/TypeStrong/tsify/issues/151#issuecomment-230092455>,
or mute
the thread
<https://github.com/notifications/unsubscribe/ABE5mAwfdEaabTqD2Hzl2vQRvoA_Momeks5qRizogaJpZM4JBYhg>
.

**Comment by cartant** [comment]
@scamden I've been going through some of the older, still-open issues.

I've added a [JS/TS](https://github.com/TypeStrong/tsify/tree/master/examples/js) example. It is essentially the same as the `greeter` example, but with a `main.js` instead of a `main.ts` and with `allowJs` option specified.

Regarding your question about where the compiled JS is written: it isn't; it's held in memory and is later emitted to Browserify.

The problems you experienced with empty modules should have been fixed in the the latest - 1.0.4 - release.
